knpu/episode3/Challenges/Parent/CallParentMethodCoding.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Challenges\Parent;
+
+use KnpU\ActivityRunner\Activity\CodingChallenge\CodingContext;
+use KnpU\ActivityRunner\Activity\CodingChallenge\CorrectAnswer;
+use KnpU\ActivityRunner\Activity\CodingChallengeInterface;
+use KnpU\ActivityRunner\Activity\CodingChallenge\CodingExecutionResult;
+use KnpU\ActivityRunner\Activity\CodingChallenge\FileBuilder;
+use KnpU\ActivityRunner\Activity\Exception\GradingException;
+
+class CallParentMethodCoding implements CodingChallengeInterface
+{
+    /**
+     * @return string
+     */
+    public function getQuestion()
+    {
+        return <<<EOF
+It took too long to travel to planets to destroy them
+in the first DeathStar, so Darth wants the laser range
+on the new DeathStar to be *twice* as far! Override
+the `getLaserRange()` method in `DeathStarII` to this
+happen. But don't repeat the `2000000` value! Call the
+parent function!
```

Comment:
>
Call the parent function and then multiple that value by 2.

Then we can remove the next sentence (Make the ...)